forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -46,6 +46,8 @@ public class GitHubPullRequest implements PullRequest {
 
     private List<Label> labels = null;
 
+    private static final int GITHUB_PR_COMMENT_BODY_MAX_SIZE = 20000;
```

Comment:
I think we can bump this up a bit more. Maybe 64k is a reasonable number? Also consider using `_` to group long number constants into triplets for easier reading.

Reply:
> I think we can bump this up a bit more. Maybe 64k is a reasonable number? Also consider using `_` to group long number constants into triplets for easier reading.

64K would work well for both GitHub and GitLab